### PR TITLE
Fixed launch guide in both b2b and b2c guidance

### PR DESCRIPTION
**Pull request by zamd**
- Added launch preperation to both b2b/b2c sections
- Fixed B2B "Keep Reading" section by including missing links
- Refactored "Keep Reading" content to remove duplicate

### CODE REVIEW

File: articles/architecture-scenarios/_includes/_keep-reading.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<% if (platform === "b2b") { %>
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_keep-reading.md]: I would kill this and just use the platform to do the links, see below...

File: articles/architecture-scenarios/_includes/_keep-reading.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+<!-- Exclude current page from keep reading section -->
+<% if (self !== "architecture") { %>
+* [Architecture](/architecture-scenarios/implementation/b2b/b2b-architecture)
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_keep-reading.md]:
```suggestion
* [Architecture](/architecture-scenarios/implementation/${platform}/${platform}-architecture)
```

File: articles/architecture-scenarios/_includes/_keep-reading.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+<% } %>
+<% if (self !== "operations") { %>
+* [Operations](/architecture-scenarios/implementation/b2b/b2b-operations)
+<% } %>
+<% if (self !== "launch") { %>
+* [Launch Preparation](/architecture-scenarios/implementation/b2b/b2b-launch)
+<% } %>
+<%  } %>
+
+<% if (platform === "b2c") { %>
```
Comment (mostekcm) [inline articles/architecture-scenarios/_includes/_keep-reading.md]: Once done making the above change, you can kill this whole block...

**Review by mostekcm — CHANGES_REQUESTED** [review]

File: articles/architecture-scenarios/_includes/_keep-reading.md (hunk 1 above)
Comment (zamd) [reply articles/architecture-scenarios/_includes/_keep-reading.md]: yes, I have already battled this for couple of hours yesterday. The trouble is our  docs "Compiler" doesn't work correctly with interpolation and markdown links. 
It shows links if I use <%=platform%> but doesn't replace with the value. 
If I introduce space as <%= platform %>, it  does the interpolation but then dont' recognize the "line" as a markdown link. 


I didn't want to debug and fix the compiler so I duplicated the IF "platform" check and removed "platform" interpolation.

File: articles/architecture-scenarios/_includes/_keep-reading.md (hunk 2 above)
Comment (zamd) [reply articles/architecture-scenarios/_includes/_keep-reading.md]: Does `${platform}` interpolation syntax work for lodash template? I thought it only supported limited ejs syntax i.e. `<%=platform%>`.

File: articles/architecture-scenarios/_includes/_keep-reading.md (hunk 3 above)
Comment (zamd) [reply articles/architecture-scenarios/_includes/_keep-reading.md]: As I explained in first comment, interpolation and markdown links doesn't play well with our docs "Compiler". That's why I chosen this fallback option.

File: articles/architecture-scenarios/_includes/_keep-reading.md (hunk 2 above)
Comment (zamd) [reply articles/architecture-scenarios/_includes/_keep-reading.md]: Did a quick "isolated test" and `${variable}` **does** work in lodash... 
Let me try in docs and update the PR.

Comment (zamd) [reply articles/architecture-scenarios/_includes/_keep-reading.md]: @mostekcm please review the last commit and approve....

**Review by mostekcm — APPROVED** [review]
LGTM!

**Pull request merged**